src/bindings/overlay.js

```diff
@@ -34,7 +35,8 @@ export function overlayElement(element, translation) {
     } else {
       // start with an inert template element and move its children into
       // `element` but such that `element`'s own children are not replaced
-      const tmpl = element.ownerDocument.createElement('template');
+      const tmpl =
+        document.createElementNS('http://www.w3.org/1999/xhtml', 'template');
```

Comment:
the problem is that XUL doesn't have ownerDocument and we don't want to create XUL element in XUL. We want to create HTML element in XUL :)